util/volume.go

```diff
@@ -0,0 +1,71 @@
+package util
+
+import (
+	"encoding/json"
+	"github.com/hpe-storage/common-host-libs/logger"
+	"github.com/hpe-storage/common-host-libs/model"
+)
+
+func GetVolumeObject(serialNumber, lunID string) *model.Volume {
+
+	var volObj *model.Volume
+	volObj.SerialNumber = serialNumber
+	volObj.LunID = lunID
+
+	return volObj
+}
+
+func GetSecondaryArrayLUNIds(details string) []int32 {
+	var secondaryArrayDetails model.SecondaryBackendDetails
```

Comment:
We have this logger pattern that we use in many of our functions...
>>>>> function name
<<<<< defer function name

to make debugging easier.  Can you do the same here?

Reply:
i will incorporate this